portal-api/src/v1/controllers/utilisation-report-service/due-report-dates.controller.js

```diff
@@ -29,19 +29,19 @@ const getNextDueReportDate = (latestReport, currentDueReportDate) => {
     return currentDueReportDate;
   }
 
-  const { month: oneIndexedMonth, year } = latestReport;
-  const latestReportDate = new Date(year, oneIndexedMonth - 1);
+  const { month: oneIndexedStartMonth, year: startYear } = latestReport.reportPeriod.start;
+  const latestReportDate = new Date(startYear, oneIndexedStartMonth - 1);
   return addMonths(latestReportDate, 1);
 };
 
 /**
  * @typedef {Object} DueReportDate
```

Comment:
If we add the generic `MonthAndYear` type to this project we just use that instead and remove this.